x-pack/test/licensing_plugin/config.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { FtrConfigProviderContext } from '@kbn/test/types/ftr';
+import { services, pageObjects } from './services';
+
+const license = 'basic';
+
+export default async function({ readConfigFile }: FtrConfigProviderContext) {
+  const functionalTestsConfig = await readConfigFile(require.resolve('../functional/config.js'));
```

Comment:
functional tests because we are going to use them for https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/xpack_main/public/hacks/check_xpack_info_change.js in the following PR